aexpect/client.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
         self.a_id = a_id or data_factory.generate_random_string(8)
         self.log_file = None
         self.closed = False
+        # Use PYTHONENCODINGS or utf-8 (instead of ascii)
+        self.encoding = os.environ.get("PYTHONENCODING", "utf-8")
```

Comment:
I guess the problem here is that setting instance.encoding is not very visible. I guess we need to create API documentation for aexpect ASAP...